.idea/checkstyle-idea.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Did you intend to check this directory in?

Reply:
> Did you intend to check this directory in?

Yes so that the spring-cloud styling is available in the project.

Reply:
Hmm. Does it just work or do you have to do something to get it to work? Typically these are not checked in.

Reply:
The same for vscode is added.
It saves a huge amount of time to get the project configured for formatting and checkstyle.